store/postgres/job_dependency_repository_test.go

```diff
@@ -0,0 +1,93 @@
+//go:build !unit_test
+// +build !unit_test
+
+package postgres
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/odpf/optimus/models"
+	"github.com/stretchr/testify/assert"
+	"gorm.io/gorm"
+)
+
+func TestJobDependencyRepository(t *testing.T) {
```

Comment:
The name of function should be `TestIntegrationJobDependencyRepository`, we use name TestIntegration to identify the integration tests.

Reply:
updated